course/course.pot

```diff
@@ -0,0 +1,390 @@
+# Copyright (C) 2023 Automattic
```

Comment:
Typically we do not commit the .pot files here, they just live on wpcom. Is there a reason to include them here?